assets/js/utils/rtcUtils.ts

```diff
@@ -294,22 +307,50 @@ const filterDevices = (allDevices: MediaDeviceInfo[], type: string) => {
   return allDevices.filter((device) => device.deviceId != "default" && device.kind == type);
 };
 
-const addOrReplaceTrack = (
+export const addOrReplaceTrack = (
```

Comment:
I would suggest passing to this function presenterStream instead of setPresenterStream and return updated value so invoking this function would look like this:
``` js
# inside setter
const updated_prev = addOrReplaceTrack(client, newCameraStream.getTracks()[0], prev);

# in regular function
setPresenterStreams( (prev) => addOrReplaceTrack(client, newCameraStream.getTracks()[0], prev);)

```

Reply:
I agree with @Karolk99